lib/plugins/aws/lib/naming.js

```diff
@@ -248,4 +254,8 @@ module.exports = {
   getLambdaAlexaSkillPermissionLogicalId(functionName) {
     return `${this.getNormalizedFunctionName(functionName)}LambdaPermissionAlexaSkill`;
   },
+  getLambdaCloudWatchLogPermissionLogicalId(functionName, logsIndex) {
+    return `${this.getNormalizedFunctionName(functionName)}LambdaPermissionCloudWatchLog${
```

Comment:
This should be renamed to `{functionName}LambdaPermissionLogsSubscriptionFilterCloudWatchLog{logsIndex}` so that it's consistent with the `cloudwatchEvent` event permission logical id (Line 246 in this file).

See: https://serverless.com/framework/docs/providers/aws/guide/resources/